### DISCUSSION TITLE
奇怪了，为什么离线词库过滤不生效呢？

### DISCUSSION THREAD

**Discussion by white0dew** [ArtalkJS/Artalk#1026] [Q&A]
我的设置如下：离线词库如下：

然后我用一个测试账号发送留言:考题作废，怎么还是通过了呢？

**Comment by white0dew** [comment]
图片漏了：

![image](https://github.com/user-attachments/assets/5ac6f0e1-30c6-4ac5-86bb-ff8c9f54566b)

配置：
![image](https://github.com/user-attachments/assets/d55b026e-e804-4c9f-adac-ca92bcff2623)

大家知道怎么回事吗

**Comment by QingYu-Su** [comment]
我在部署时也遇到了这个问题，经过尝试后，我的解决方案是：在artalk的配置中不使用“\n”作为分割符，而是使用"|"。相应的，词库文件中每个词之间也需要使用"|"隔开，然后就可以正常使用离线词库过滤了。
